meta/Decays.m

```diff
@@ -2238,7 +2238,8 @@ that returns a total (sumed over internal insertions) 1-loop amplitude for a giv
                  AbsoluteTiming@ParallelMap[
                     CreateTotalAmplitudeSpecialization[#, modelName]&,
                     Flatten[Last @@@ particleDecays, 1],
-                    DistributedContexts -> All, Method -> "FinestGrained"
+                    DistributedContexts -> None,
```

Comment:
With `DistributedContexts -> None` the calculation is not parallelized.

Reply:
Yes, now I see it. Thank you! Let me look at more carefully.